### PR TITLE
 Add reporting tools recipe

### PR DESCRIPTION
**Pull request by robbieaverill**

### CODE REVIEW

**Comment by madmatt** [comment]
Has there already been some discussion about the naming of these recipes?

I expect the number of these recipes would increase over time, and it seems like it will make it harder to browse through the `silverstripe` account (even if they all start with `recipe-`).

Has any thought been given to creating a separate account just for recipes - e.g. `silverstripe-recipes/reporting-tools`? Seems a little cleaner to me to keep recipes separate from modules and the core.

Just my 2c at least, figured I'd drop it here, but if there's a better place to put it just let me know!

**Comment by robbieaverill** [comment]
Yeah we could do that! Also could use GitHub tagging to categorise them. cc @tractorcow

**Comment by tractorcow** [comment]
I haven't seen much community practice in "multi-vendoring", so I'm hesitant to recommend it ourselves.

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "name": "silverstripe/recipe-reporting-tools",
+    "type": "silverstripe-recipe",
+    "description": "Add extra CMS reporting tools to your SilverStripe project",
+    "homepage": "https://www.silverstripe.org",
+    "license": "BSD-3-Clause",
+    "require": {
+        "silverstripe/recipe-plugin": "^1",
+        "silverstripe/recipe-cms": "1.1.x-dev",
+        "silverstripe/externallinks": "2.x-dev",
+        "silverstripe/reports": "4.x-dev",
```
Comment (tractorcow) [inline composer.json]: Reports is in recipe-cms, can be removed.

**Pull request merged**